README.md

```diff
@@ -1,3 +1,150 @@
+# The BlueBrain Spack Deployment
+
+Official documentation [below](#-spack).
+For the development documentation of the deployment stack, see
+[`deploy/README.md`](deploy/README.md).
+
+## Building software with Spack at BlueBrain
+
+On BB5, clone this repository to get started using Spack.
+The following commands are a good way to get started:
+
+    $ git clone git@github.com:BlueBrain/spack.git
+    $ . spack/share/spack/setup-env.sh
+    $ spack spec -I spykfunc|head -n 15
+    Input spec
+    --------------------------------
+     -   spykfunc
+
+    Concretized
+    --------------------------------
+     -   spykfunc@0.12.0%gcc@4.8.5 arch=linux-rhel7-x86_64
+     -       ^hdf5@1.10.4%gcc@4.8.5~cxx~debug~fortran+hl~mpi+pic+shared~szip~threadsafe arch=linux-rhel7-x86_64
+     -           ^zlib@1.2.11%gcc@4.8.5+optimize+pic+shared arch=linux-rhel7-x86_64
+     -       ^highfive@1.6%gcc@4.8.5~boost build_type=RelWithDebInfo ~mpi arch=linux-rhel7-x86_64
+     -           ^cmake@3.13.0%gcc@4.8.5~doc+ncurses+openssl+ownlibs~qt arch=linux-rhel7-x86_64
+     -               ^ncurses@6.1%gcc@4.8.5~symlinks~termlib arch=linux-rhel7-x86_64
+     -                   ^pkgconf@1.5.4%gcc@4.8.5 arch=linux-rhel7-x86_64
+     -               ^openssl@1.1.1%gcc@4.8.5+systemcerts arch=linux-rhel7-x86_64
+     -                   ^perl@5.26.2%gcc@4.8.5+cpanm patches=0eac10ed90aeb0459ad8851f88081d439a4e41978e586ec743069e8b059370ac +shared+threads arch=linux-rhel7-x86_64
+
+Here we see all the software that would be required to build one program of
+the circuit building workflow.
+The leading `-` sign in the output signifies that this particular piece of
+software would have to be built from scratch.
+To reduce the amount of time spent compiling the same software, we can
+configure Spack to use centrally build packages on BB5:
+
+    $ mkdir -p ~/.spack
+    $ ln -s /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/{compilers,packages,upstreams}.yaml ~/.spack
+    $ spack spec -I spykfunc|head -n 15
```

Comment:
could you add instructions to set INSTALL_PATH?

Reply:
Done